### PR TITLE
doc: fix incorrect testmempoolaccept doc

### PR DESCRIPTION
**Pull request by glozow**
Sorry, I somehow missed this...

### CODE REVIEW

File: src/rpc/rawtransaction.cpp [hunk 1]
```diff
@@ -894,8 +894,7 @@ static RPCHelpMan testmempoolaccept()
                 "\nThis checks if transactions violate the consensus or policy rules.\n"
                 "\nSee sendrawtransaction call.\n",
                 {
-                    {"rawtxs", RPCArg::Type::ARR, RPCArg::Optional::NO, "An array of hex strings of raw transactions.\n"
-            "                                        Length must be one for now.",
+                    {"rawtxs", RPCArg::Type::ARR, RPCArg::Optional::NO, "An array of hex strings of raw transactions.\n",
```
Comment (jnewbery) [inline src/rpc/rawtransaction.cpp:897]: I don't think we need this line break:

```suggestion
                    {"rawtxs", RPCArg::Type::ARR, RPCArg::Optional::NO, "An array of hex strings of raw transactions.",
```

**Review by jnewbery — COMMENTED** [review]
Concept ACK

File: src/rpc/rawtransaction.cpp (hunk 1 above)
Comment (glozow) [reply src/rpc/rawtransaction.cpp:897]: Done

**Comment by jnewbery** [comment]
Tested ACK 9169be09f49c82fece034285e92f8ffa41e19ee2

**Pull request merged**